chapter_13_database_layer_validation.asciidoc

```diff
@@ -695,13 +695,13 @@ This will immediately break our original functional test, because the
 ----
 $ pass:quotes[*python src/manage.py test functional_tests*]
 [...]
-selenium.common.exceptions.NoSuchElementException: Message: Unable to locate
-element: .invalid-feedback; [...]
-[...]
 AssertionError: '2: Use peacock feathers to make a fly' not found in ['1: Buy
```

Comment:
a nice side-effect of this fix - the book tests failed on this bit, saying it couldn't find this part of the traceback.  sure enough, it's because this error was coming from the FT that we've put an early return in, so it doesn't actually appear any more.

this is exactly what the book tests are for, yay!

Reply:
Oh that's a nice catch!